lib/pluginUtils.js

```diff
@@ -104,7 +104,8 @@ function checkVersion (fn) {
   const requiredVersion = meta.fastify
 
   const sanitizedVersion = this.version.replace(/-rc.\d+$/, '')
-  if (requiredVersion && !semver.satisfies(sanitizedVersion, requiredVersion)) {
+  const fastifyRc = /-rc.+$/.test(this.version)
```

Comment:
should not be possible to remove the `.replace` statement by using the `includePrerelease` flag?